### PR TITLE
docs: Make the main README a great resource for all Camp Content

### PR DESCRIPTION
**Pull request by daviddias**
WIP:
- [x] 👩🏽‍🏫 Core & Elective Courses
- [ ] 📃 Poster Projects
- [x] 🐋 Deep Dives
- [x] ⚡️ Lightning Talks
- [x] 🧬 Sci-Fi Fair
- [x] 🧩 Unconf
- [x] 🐚 Quiet Room

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,13 +13,88 @@ If you missed IPFS Camp or want to revisit its content, everything will remain a
...
+| Starting, Growing, and Stewarding your IPFS Community              | @nukemandan @kk3wong @stefanhans     | Let's get Meta! A workshop on effective workshops! And how to foster a community                                                                | [ELECTIVE_COURSE_F](https://github.com/ipfs/camp/tree/master/CORE_AND_ELECTIVE_COURSES/ELECTIVE_COURSE_F) | soon™ |
+| Managing Datasets with QRI                                         | @b5                                  | An open source tool for versioned, scriptable, exportable, collaborative datasets | [ELECTIVE_COURSE_G](https://github.com/ipfs/camp/tree/master/CORE_AND_ELECTIVE_COURSES/ELECTIVE_COURSE_G) | soon™ |
+| Making the Browser a True User Agent                               | @autonome @Gozala @soapdog           | We’ll show how browsers act as a value filter on our world, teach some approaches to making the browser truly be an agent that represents you, and break into groups to start building our own true user agents.| [ELECTIVE_COURSE_H](https://github.com/ipfs/camp/tree/master/CORE_AND_ELECTIVE_COURSES/ELECTIVE_COURSE_H) | soon™ |
+
+### [📃 Poster Projects](POSTER_PROJECTS)
```
Comment (daviddias) [inline README.md:35]: @djdv can you help me understand which Posters Projects happened and where are the photos of the posters?

File: README.md [hunk 2]
```diff
@@ -13,13 +13,88 @@ If you missed IPFS Camp or want to revisit its content, everything will remain a
...
+
+TODO
+
+### [🐋 Deep Dives](DEEP_DIVES)
```
Comment (daviddias) [inline README.md:39]: @alanshaw can you help me understand which Deep Dives happened? I want to link them form the main README and then soon link the recording too

File: README.md [hunk 3]
```diff
@@ -13,13 +13,88 @@ If you missed IPFS Camp or want to revisit its content, everything will remain a
...
+| Textile Photos    | @andrewxhill, @carsonfarmer, @balupton | a decentralized photo booth | soon™ |
+| Lazer Catbot      | @gorhgorh | 😼 | soon™ |
+| FAPRS             | @MarneeDear | IPFS over Ham Radio | soon™ |
+| Brave             | @yrliou, @darkdh, @emerick | native IPFS in Brave browser! | soon™ |
+| ENS               | @makoto | resolving ENS/IPFS with DNS | soon™ |
+| IPFS Spider       | @magnshen | crawling the IPFS network | soon™ |
+| Cluster Labs Horizon + more | @vasa-develop | viz for the RPi cluster + more | soon™ |
+| libp2p-ble-transport | @aeddi | IPFS over other kinds of radio! | soon™ |
+
+### [🧩 Unconf](UNCONF)
```
Comment (daviddias) [inline README.md:122]: @jessicaschilling do we have a record of which Unconf sessions happened and their respective notes?

File: README.md (hunk 2 above)
Comment (alanshaw) [reply README.md:39]: All of them listed here happened https://github.com/ipfs/camp/tree/master/DEEP_DIVES#topics apart from 23 - connection management.

**Comment by daviddias** [comment]
@doctorrobinson fyi, I'm creating a single README for all the created content during this event as you can see here -- https://github.com/ipfs/camp/blob/89a1a96f74f301bbe6f7fc4ca66809cb249178b3/README.md. My hopes is as you publish the videos at #141, I add the url to the specific cell on the Recording column :)

File: README.md (hunk 2 above)
Comment (daviddias) [reply README.md:39]: Thank you Sir! :)

**Comment by daviddias** [comment]
Going for the merge to avoid conflicts and to start making discoverability easier

File: README.md (hunk 1 above)
Comment (djdv) [reply README.md:35]: @daviddias 
I believe video and image media is on the Google drive but has yet to be processed.
Some of the sessions had no attendees so we should double check with the full topic list and the media we have. (presentations and pictures)

@doctorrobinson @yunigraham 
Have the photos of the actual posters themselves been processed?
Likewise with the presentation videos. (I know there may be issues due to the power outage)

Comment (doctorrobinson) [reply README.md:35]: Video Updates here: https://github.com/protocol/video/issues/55

**Pull request merged**